orbit-2-rxjava2/README.md

```diff
@@ -0,0 +1,87 @@
+# Orbit 2 RxJava2 plugin
+
+The RxJava 2 plugin provides:
+
+- RxJava 2 DSL operators
+
+## Including the module
+
+```kotlin
+implementation("com.babylon.orbit2:orbit-rxjava2:<latest-version>")
+```
+
+## RxJava 2 DSL Operators
+
+The Core DSL contains the following operators:
+
+- transformObservable
+- transformSingle
+- transformMaybe
+- transformCompletable
+
+### transformObservable
+
+``` kotlin
+class ExampleViewModel : ContainerHost<ExampleState, ExampleSideEffect> {
+    ...
+
+    fun startLocationTracking() = orbit {
+            transformObservable { subscribeToLocationUpdates() }
+                .reduce { state.copy(currentLocation = event) }
+        }
+    }
+}
+```
+
+You can use this operator to subscribe to hot or cold observables.
+This operator acts similar to `flatMap`.
+
+### transformSingle
+
+``` kotlin
+class ExampleViewModel : ContainerHost<ExampleState, ExampleSideEffect> {
+    ...
+
+    fun example() = orbit {
+            transformSingle { apiCall() }
+                .transformSingle { anotherApiCall(event) } // Use the result of the first api call
+        }
+    }
+}
+```
+
+You can use this operator to subscribe to an RxJava 2 `Single`.
+This operator acts similar to `flatMapSingle`.
+
+### transformMaybe
+
+``` kotlin
+class ExampleViewModel : ContainerHost<ExampleState, ExampleSideEffect> {
+    ...
+
+    fun example() = orbit {
+            transformMaybe { getLoggedInUser() }
```

Comment:
regarding this example and most of the others, we don't see what the syntax of the functions are, i.e. what type does getLoggedInUser return? perhaps more relevant for coroutine code as its less obvious if a function is suspending or not

Reply:
I realise it gets stated below but I think this is too late